### ISSUE TITLE
no-invalid-template-strings  not working

### ISSUE THREAD

**Issue by ShayBox** [palantir/tslint#3815]
### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__: 2.8.1
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
console.log(`Test`);
```

with `tslint.json` configuration:

```json
{
	"extends": "tslint:recommended",
	"rules": {
		"no-console": false,
		"no-invalid-template-strings": true
	}
}
```

#### Actual behavior

No errors

#### Expected behavior

Errors from using template strings without ${} in them

And yes, other rules are working

**Comment by JoshuaKGoldberg** [comment]
> Errors from using template strings without ${} in them

What are the errors?

**Comment by ShayBox** [comment]
None, that's the expected behavior, that's the problem

**Comment by ShayBox** [comment]
Any news on this getting fixed? Annoying having unneeded backticks in code and the rule not warning me about them.

**Comment by johnwiseheart** [comment]
Will accept a PR to fix this issue.

**Comment by JoshuaKGoldberg** [comment]
@ShayBox the rule that's intended to complain about this is **`quotemark`**. Have you tried enabling that rule?

`no-invalid-template-strings` is meant to go the other way: it should complain about non-template strings that have `${}` in them. For example:

```typescript
console.log("Test: ${value}");
```

/cc @johnwiseheart - I think this is a working-as-designed.

**Comment by ShayBox** [comment]
I have quotemark set to single, it tells me to replace doubles with singles, but it doesn't tell me to replace backticks with singles if there's no template in them, which is possible in eslint.

**Comment by JoshuaKGoldberg** [comment]
Gotcha - seems like the problem here is with the `quotemark` rule. It has an `"avoid-template"` option for this case you can use. 

```json
{
	"rules": {
		"quotemark": [true, "avoid-template"],
		"no-invalid-template-strings": true
	}
}
```
